beacon-chain/rpc/beacon/validators_test.go

```diff
@@ -32,6 +32,10 @@ import (
 	"github.com/prysmaticlabs/prysm/shared/timeutils"
 )
 
+const (
+	noEpochInfoError = "Cannot retrieve information about an epoch in the future"
```

Comment:
```suggestion
	noFutureEpochInfoError = "Cannot retrieve information about an epoch in the future"
```

Reply:
I tend to step away from suggestion comments for names. Committing this suggestion will break the build because a proper refactor won't happen.

Just my observation :-)